egs2/vctk/asr1/conf/fbank.conf

```diff
@@ -0,0 +1,2 @@
+--sample-frequency=48000 
```

Comment:
The sr seems to be very high for the recipe, is this intentional?

Reply:
meanwhile, this (and also pitch.conf) will only be used when doing kaldi style feature extraction (not the default one), so if you want to use the sampling rate for default one, you should set `fs` in the asr.sh as well

Reply:
done